components/brave_rewards/browser/rewards_service_browsertest.cc

```diff
@@ -45,6 +43,28 @@ std::unique_ptr<net::test_server::HttpResponse> HandleRequest(
   return std::move(http_response);
 }
 
+bool URLMatches(const std::string& url,
+                const std::string& path,
+                const std::string& prefix,
+                const SERVER_TYPES& server) {
+  const std::string target_url =
+      braveledger_bat_helper::buildURL(path, prefix, server);
+  return (url.find(target_url) == 0);
+}
+
+void Split(std::vector<std::string>* tmp,
+           const std::string& query,
+           char delimiter) {
+  DCHECK(tmp);
+  std::stringstream ss(query);
+  std::string item;
+  while (std::getline(ss, item, delimiter)) {
+    if (query[0] != '\n') {
```

Comment:
Would this cause the CRLF problems on windows?

Reply:
Well, I didn't test this PR on Windows yet. (I didn't modify this logic in this PR)
Will check this PR on Windows.

Reply:
👍 May not be relevant here but a couple of us have been bitten by CRLF delimiters.

Reply:
Rewards browser test works well on my local Win10. :)

Reply:
maybe just `base::SplitString`?

Reply:
Good to know this api! thanks. Done.